docs/overview.md

```diff
@@ -0,0 +1,67 @@
+# An overview
+
+nilDb is a structured data persistence service that provides a robust, distributed, and queryable data solution when multiple nodes are combined into a cluster. Users write data schemas, upload those schemas to nilDB clusters, and then perform data CRUD operations against them. Additionally, users can write aggregation pipelines to produce insights from their data. Users can apply advanced cryptography to enable secret sharing and other secure data processing use cases.
+
+## Contents
+
+- [Networks](#networks)
+- [Network topology](#network-topology)
+    - [Clusters](#clusters)
+    - [Node Architecture](#node-architecture)
+- [Clients](#clients)
+
+## Networks
+
+There are two nilDB networks:
+
+1. _Mainnet_ refers to the group of nodes integrated with nilChain mainnet (not yet launched).
+2. _Devnet_ refers to the group of nodes integrated with nilChain testnet.
```

Comment:
Stick with Testnet <> Testnet convention? Or are you dead-set on Devnet <> Testnet?